src/components/Window.js

```diff
@@ -10,19 +10,36 @@ import ThumbnailNavigation from '../containers/ThumbnailNavigation';
  * @param {object} window
  */
 export class Window extends Component {
+  /**
+   * wrappedTopBar - will conditionally wrap a WindowTopBar for needed
+   * additional functionality based on workspace type
+   */
+  wrappedTopBar() {
```

Comment:
Open to other suggestions on the best way to do this. We essentially need to conditionally wrap this component.